operator-pipeline-images/operatorcert/schemas/ci-schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "title": "ci.yaml validation schema",
+    "description": "json schema validating the ci.yaml",
+    "type": "object",
+    "additionaProperties": true,
+    "properties": {
+        "fbc": {
+            "description": "Config set for FBC-enabled operator bundles",
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "Whether the operator already have FBC support or not",
```

Comment:
Let's rephrase it.. `Flag that determines whether the operator uses FBC delivery method.` The original message might be misleading since FBC is not a feature that the operator offers.